modules/docs/arrow-docs/docs/docs/README.md

```diff
@@ -41,67 +41,22 @@ allprojects {
     repositories {
         mavenCentral()
         jcenter()
-        maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" }
+        maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" } // for SNAPSHOT builds
         maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
     }
 }
 ```
 
-# Dependency breakdown starting in Arrow 0.9.1
-
-Starting in 0.9.1, Arrow follows the following convention for artifact publication.
-
-The arrow modules are Core, Effects, Optics, Recursion, etc.
-
-An Arrow module is composed of data types and type classes.
-Arrow modules are exported and published with the following semantics.
-
-If we take for example `arrow-core`.
-
-Arrow core contains the basic arrow type classes and data types and it's composed of 2 main artifacts that may be used a la carte:
-
-Recomended for most use cases:
-
-- `arrow-core` (Depends on data and extensions modules and exports both)
-
-Trimmed down versions:
-
-- `arrow-core-data` (Only data types & typeclasses)
-- `arrow-core` (Data types, typeclasses & type class extensions)
-
-# Current stable version 0.9.0
-
-```groovy
-def arrow_version = "0.9.0"
-```
-
-You can find the dependencies necessary in the Basic Setup of the README at the 0.9.0 tag clicking [here](https://github.com/arrow-kt/arrow/blob/0.9.0/README.md#next-development-version-090).
-
-# Next development version 0.9.1
+# Next development version 0.10.0
 
 Add the dependencies into the project's `build.gradle`
 
 ```groovy
-def arrow_version = "0.9.1-SNAPSHOT"
+def arrow_version = "0.10.0-SNAPSHOT"
 dependencies {
```

Comment:
Same comment here about all missing packages, why just keeping these 3? shouldn't we add all the global ones then link to the new libraries docs for the complete list or something?

Reply:
The idea is that we'd be adding here more libs as they are promoted out of incubation and are feature complete. This is how it has been designed for the new site. I believe advertising them in the README creates more confusion as some like Free and MTL are exploratory and do not have much industrial application at the moment whereas core, fx, and optics is what we are pushing forward at the moment until others get out of their experimental status.

Reply:
But what I see here is `core`, `syntax` and `meta`. Shouldn't we include `fx` at least? `fx` is essentially all about how we want to promote people to use the library.

Reply:
Arrow-fx is a few lines down bellow.